project/CrossJava.scala

```diff
@@ -117,6 +117,17 @@ object CrossJava {
             }
     }
 
+    class WindowsDiscoverConfig extends JavaDiscoverConf {
+      val base: File = file("C://Program Files/Java")
```

Comment:
Isn't there also a different app directory for 64-bit installations on windows?

Reply:
I think `C://Program Files` is ok as follows. I couldn't find the official Microsoft resource so let me paste this random article, but as far as I checked this seems correct.

https://www.howtogeek.com/129178/why-does-64-bit-windows-need-a-separate-program-files-x86-folder/

> In summary, on a 32-bit version of Windows, you just have a “C:\Program Files” folder. This contains all your installed programs, all of which are 32-bit.

> On a 64-bit version of Windows, 64-bit programs are stored in the “C:\Program Files” folder and 32-bit programs are stored in the “C:\Program Files (x86)” folder.

Reply:
Ah ok, that's how they do it, kinda makes sense (I have just seen it at some point on someone else machine, haven't run windows for ages). Thanks for verifying.